src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.cmd

```diff
@@ -1,39 +1,29 @@
 @echo off
 setlocal
 
-set PROJECT_DIR=%~1
-set DOTNET_CMD=%~2
-set TOOLRUNTIME_DIR=%~3
-IF [%BUILDTOOLS_TARGET_RUNTIME%]==[] set BUILDTOOLS_TARGET_RUNTIME=win7-x64
+set DOTNET_CMD=%~1
+set TOOLRUNTIME_DIR=%~2
 set BUILDTOOLS_PACKAGE_DIR=%~dp0
-set PORTABLETARGETS_VERSION=0.1.1-dev
 set ROSLYNCOMPILERS_VERSION=1.2.0-beta1-20160202-02
```

Comment:
yeah, the only way to control this would be to have the Microsoft.Net.Compilers package to follow the contentFiles convention, and then we could move this dependency out of here, and put it in the nuspec as a net45 dependency, and we would get it restored there. we would still have the two versions on the nuspec, but at least they would be on the same file.